components/net/bluetooth_test.rs

```diff
@@ -118,476 +106,436 @@ pub fn set_adapter(adapter: &BluetoothAdapter, adapter_name: String, sender: &Ip
     set_attribute_or_return_error(adapter.set_discoverable(true), sender);
 }
 
-// Create Devices
+// Create Device
+pub fn create_device(adapter: &BluetoothAdapter,
+                     name: String,
+                     address: String,
+                     sender: &IpcSender<BluetoothResult<()>>)
+                     -> BluetoothDevice {
+    let device = BluetoothDevice::create_device(adapter.clone(), generate_id());
+    set_attribute_or_return_error(device.set_name(Some(name)), sender);
+    set_attribute_or_return_error(device.set_address(address), sender);
+    set_attribute_or_return_error(device.set_connectable(true), sender);
+    device
+}
 
-pub fn create_heart_rate_device(adapter: &BluetoothAdapter,
+// Create Rate Device with UUIDs
```

Comment:
Did you mean "Create Heart Rate Device with UUIDs" or "Create Device with UUIDs"?